File: P4/konversisuhu.py
# NAMA  : Ahmad Nadil
# NIM   : 16521516
# Judul : Konversi Suhu

# ALGORITMA
t = float(input())  # Besar suhu (Celcius)
k = input()         # Suhu tujuan

if k == 'R':
    print("%.2f" % ((4/5) * t))
elif k == 'F':
    print("%.2f" % ((9/5) * t + 32))
elif k == 'K':
    print("%.2f" % (t + 273.15))

File: P1/KonversiSuhu.hs
{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
-- NIM   : 16521516
-- Nama  : Ahmad Nadil
-- Judul : Konversi Suhu

-- Definisi dan Spesifikasi
module KonversiSuhu where
    konversiSuhu :: Float -> Char -> Float

-- Realisasi
    konversiSuhu t s
        | s == 'R' = 4/5 * t
        | s == 'F' = (9/5 * t) + 32
        | s == 'K' = t + 273.15

-- Aplikasi
    -- konversiSuhu 25 'R'